scripts/deploy/index.ts

```diff
@@ -17,7 +17,7 @@ async function deploy(network: NetworkProvider.Test | NetworkProvider.Main) {
     console.log('*** UPDATE YOUR .ENV ***');
 }
 
-const arg = process.argv[2];
```

Comment:
This worked fine, please revert change :)

Reply:
Ah that was a little local hack:P